dask/bytes/tests/test_s3.py

```diff
@@ -447,6 +449,13 @@ def test_parquet(s3, engine, s3so):
     lib = pytest.importorskip(engine)
     if engine == "pyarrow" and LooseVersion(lib.__version__) < "0.13.1":
         pytest.skip("pyarrow < 0.13.1 not supported for parquet")
+    if (
+        engine == "pyarrow"
+        and LooseVersion(lib.__version__) >= "2.0"
+        and LooseVersion(lib.__version__) < "3.0"
```

Comment:
you can do "== 2.0.0" since we didn't do a bug fix release (and are not planning anymore, since 3.0 is coming)